src/main/scala/tilelink/RAMModel.scala

```diff
@@ -34,7 +34,7 @@ class TLRAMModel(log: String = "", ignoreCorruptData: Boolean = false, ignoreDen
       val maxTransfer  = edge.manager.maxTransfer
       val beatBytes    = edge.manager.beatBytes
       val endAddressHi = (endAddress / beatBytes).intValue
-      val maxLgBeats   = log2Up(maxTransfer/beatBytes)
+      log2Up(maxTransfer/beatBytes)
```

Comment:
Another val removal that makes the code more confusing, though I agree the whole line could be removed

Reply:
Scalafix tracks dangling references. In this line, it (potentially) saved on a JVM alloc, but couldn't remove the same line due to a potential side effect.

So, as I see, it removes `val` decl, but keeps such calls as `Unit` references in case of side-effecting calls, which Scalafix seems not be able to discover at this moment